### DISCUSSION TITLE
can not install foundry

### DISCUSSION THREAD

**Discussion by Amio13** [Cyfrin/foundry-full-course-cu#825] [General]
Can't install Foundry
Anytime I run the command

```
curl -L https://foundry.paradigm.xyz | bash

```

after few seconds of trying to load
 I get 

```
could not resolve host :foundry.paradigm.xyz

```

**Comment by debay1010** [comment]
In what environment are you getting this error? - mac, linux, vm, WSL ?

Also this seems like a dns problem. 
Try and change your dns server address to 8.8.8.8. This is the google dns server address and see if it solves your problem

**Reply by Amio13** [reply]
Thanks solved
